pkg/builtin/types.go

```diff
@@ -43,4 +43,5 @@ const (
 	Month
 	Weekday
 	Lpad
+	Acos
```

Comment:
Move Acos together with sin/cos/tan since they are families

By the way, move Lpad together with LTrim/RTrim due to the same reason

Reply:
Done